CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing to RudderStack
+
+Thanks for taking the time and for your help in improving this project!
+
+## Table of contents
+
+- [**RudderStack Contributor Agreement**](#rudderstack-contributor-agreement)
+- [**Contribute to this project**](#contribute-to-this-project)
+- [**Submitting a pull request**](#submitting-a-pull-request)
```

Comment:
@ameypv-rudder we don't have description regarding this section below. Can you add some description below?

Reply:
Sure - fixed this.